src/main/java/tr/com/srdc/cda2fhir/validation/ValidatorImpl.java

```diff
@@ -105,97 +133,132 @@ public OutputStream validateBundle(Bundle bundle) {
 		} catch (IOException e) {
 			logger.error("Could not write to the output stream.", e);
 		}
-		
+
 		return outputStream;
 	}
-	
+
+	public void logValidationResults(List<ValidationResult> results) {
```

Comment:
I won't be using this logging function in Higgs, FYI. The results should be logged at the warn level over there.

Reply:
gotchya. I just removed the function.